src/OpenTK.Mathematics/MathHelper.cs

```diff
@@ -67,6 +67,69 @@ public static class MathHelper
         /// </summary>
         public const float Log2E = 1.442695041f;
 
+        /// <summary>
+        /// Returns the sine of the specified angle.
+        /// </summary>
+        /// <param name="a">An angle, measured in radians.</param>
+        /// <returns>Sine of the angle. If a is equal to NaN, NegativeInfinity, or PositiveInfinity, this method returns NaN.</returns>
+        public static double Sin(double a) => Math.Sin(a);
+
+        /// <summary>
+        /// Returns the hyperbolic sine of the specified angle.
+        /// </summary>
+        /// <param name="a">An angle, measured in radians.</param>
+        /// <returns>Hyperbolic sine of the specified angle. If a is equal to NaN, NegativeInfinity, or PositiveInfinity, this method returns NaN.</returns>
+        public static double HSin(double a) => Math.Sinh(a);
+
+        /// <summary>
+        /// Returns the arc sine of the specified angle.
+        /// </summary>
+        /// <param name="a">An angle, measured in radians.</param>
+        /// <returns>Arc sine of the specified angle. If a is equal to NaN, NegativeInfinity, or PositiveInfinity, this method returns NaN.</returns>
+        public static double ASin(double a) => Math.Asin(a);
```

Comment:
As arcsine is a single word and is often depicted as asin in math, I guess Sin should be lowercase as well->Asin(like in System.Math).
Other opinions on that?

Reply:
I agree.

Reply:
Fixed in most recent commit